edx_sga/tests.py

```diff
@@ -240,13 +295,17 @@ def test_student_view_with_score(self, Fragment, render_template):
         fragment.initialize_js.assert_called_once_with(
             "StaffGradedAssignmentXBlock")
 
+    # pylint: disable=unused-argument
```

Comment:
All of these need to be inside the function declaration, or next to it.  Putting it above scopes the message disabling for the entire module. So for example, you could leave just the first one and all the others would be redundant.